app/assets/javascripts/contacts.js

```diff
@@ -72,8 +85,101 @@ var processContacts = function(contacts, status, xhr) {
   });
 };
 
-var showContactInForm = function(position) {
+var showEmptyContactList = function() {
+  $(".contacts").append(
+    "<li data-role='contact'>No contacts yet</li>"
+  );
+};
+
+var processContacts = function(contacts, status, xhr) {
+  storedContacts = contacts;
+  $("[data-role='contact']").remove();
+
+  if (anyContacts()) {
```

Comment:
I think I would move the concept of showing an empty or full contact list to its own function.